src/main/scala/zio/prelude/coherent/coherent.scala

```diff
@@ -211,7 +191,27 @@ object DeriveEqualTraversable {
     }
 }
 
-trait EqualIdentity[A] extends AssociativeEqual[A] with Identity[A]
+trait EqualAssociative[A] extends Equal[A] with Associative[A]
```

Comment:
Should be `AssociativeEqual`, right?

Reply:
I wanted to unify the naming scheme. So you're saying that you would prefer `XxxEqual`? Should I rename `EqualIdentity` to `IdentityEqual`?

Reply:
At least the way I had tried to unify the naming scheme was it always listed the type classes in alphabetical order. So it would be AssociativeEqual and EqualIdentity.

Reply:
Oh, like that, OK